extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderController.class.php

```diff
@@ -8,6 +8,22 @@ public function getAssets() {
 		$response = $this->getResponse();
 		$response->setFormat( WikiaResponse::FORMAT_JSON );
 		$response->setVal( 'css', AssetsManager::getInstance()->getURL( 'portable_infobox_scss' ) );
+
+		$params = $this->getRequest()->getParams();
+		if ( isset( $params['title'] ) ) {
+			$infoboxes = PortableInfoboxDataService::newFromTitle(
+				Title::newFromText($params['title'],NS_TEMPLATE)
+			)->getInfoboxes();
+
+			$builderService = new PortableInfoboxBuilderService();
+			if ( $builderService->isValidInfoboxArray( $infoboxes ) ) {
+				$response->setVal( 'data', $builderService->translateMarkupToData( $infoboxes[0] ) );
```

Comment:
if there are none or there is more then one, the above condition evaluates to false